programs/ssl/ssl_client2.c

```diff
@@ -1728,7 +1737,7 @@ int main( int argc, char *argv[] )
     fflush( stdout );
 
     len = mbedtls_snprintf( (char *) buf, sizeof( buf ) - 1, GET_REQUEST,
-                            opt.request_page );
+                            opt.request_page, opt.server_name );
```

Comment:
When we move the header change to a separate commit, this `mbedtls_snprintf()` change needs to move to that same commit.